changelog/7-33-0-Release.md

```diff
@@ -45,7 +45,7 @@
 * [org.openrewrite.java.migrate.jakarta.JavaxJwsToJakartaJws](https://docs.openrewrite.org/reference/recipes/java/migrate/jakarta/javaxjwstojakartajws): Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
 * [org.openrewrite.java.migrate.jakarta.JavaxMailToJakartaMail](https://docs.openrewrite.org/reference/recipes/java/migrate/jakarta/javaxmailtojakartamail): Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
 * [org.openrewrite.java.migrate.jakarta.JavaxMigrationToJakarta](https://docs.openrewrite.org/reference/recipes/java/migrate/jakarta/javaxmigrationtojakarta): Jakarta EE 9 is the first version of Jakarta EE that uses the new `jakarta` namespace.
-* [org.openrewrite.java.migrate.jakarta.JavaxPeristenceXmlToJakartaPersistenceXml](https://docs.openrewrite.org/reference/recipes/java/migrate/jakarta/javaxperistencexmltojakartapersistencexml):
+* [org.openrewrite.java.migrate.jakarta.JavaxPersistenceXmlToJakartaPersistenceXml](https://docs.openrewrite.org/reference/recipes/java/migrate/jakarta/javaxpersistencexmltojakartapersistencexml):
```

Comment:
It's possible this refers to the deleted thing, but, if it does, it's a dangling link

Reply:
Noticed that's been fixed in rewrite-migrate-java, so it'll work the next time we generate the recipe markdown. Thanks for pointing it out!